logger/info.log

```diff
@@ -0,0 +1,11 @@
+{"level":"info","message":"Listening on port 3000","timestamp":"2022-02-24T08:13:28.188Z"}
```

Comment:
put it in gitignore

Reply:
Didn't find any valid solution which will solve this issue. 
I have used 
- **/*.log
- *.log

Reply:
So here the issue is, you have already committed this file once, and now search how to put a committed file in gitignore. You remove the file from git tree.

Reply:
Untracked logger files